test/base/test_logging.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+
+import os
+
+import logging
+import logging.config
+
+# This is done in paster or galaxy.main for server app, provide
+# an entry point testing as well.
+logging_config = os.environ.get("GALAXY_TEST_LOGGING_CONFIG", None)
+logging_config_file = None
+if logging_config is not None:
```

Comment:
If `logging_config` is the empy string (i.e. after `export GALAXY_TEST_LOGGING_CONFIG=`), this test will pass, resulting in `logging_config_file` being equal to `os.getcwd()`.